src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -317,6 +340,18 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
                         min={0}
                         max={15}
                         defaultValue={this.props.computationPayload.election.threshold}
+                        originalValue={this.props.settingsPayload.comparison.electionThreshold}
```

Comment:
I suppose this goes under the SmartNumericInput change, but I am still unsure what originalValue actually does or how it benefits.